system_tests/program_test.go

```diff
@@ -884,18 +884,18 @@ func TestProgramAcivationLogs(t *testing.T) {
 	if len(receipt.Logs) != 1 {
 		Fatal(t, "expected 1 log while activating, got ", len(receipt.Logs))
 	}
-	parsed, err := arbWasm.ParseProgramActivated(*receipt.Logs[0])
+	log, err := arbWasm.ParseProgramActivated(*receipt.Logs[0])
```

Comment:
log is a bad name for a var.. in most of our codebase log is geth's log module name.

Reply:
That's a reasonable argument, though let's keep it for now only because we use that name elsewhere in this folder when we have logs.